docs/UserGuide.md

```diff
@@ -310,41 +421,57 @@ Format: `delete-tag t/TAGNAME [f/]`
 * The `Tag` cannot be assigned to any contacts.
 * If the `Tag` is in used, you can use `f/` to force the deletion of the `Tag` and unassign this tag from all contacts.
 
-## Wedding Features
+---
+<br>
 
-### Adding a Wedding : `create-wedding`
+<h3 class="features">Wedding Features</h3>
+
+##### Adding a Wedding : `create-wedding`
 
 Creates a `Wedding` within WedLinker to be with contacts.
 
 Format: `create-wedding w/WEDDINGNAME`
 
 * The `WEDDINGNAME` is alphanumeric and can contain whitespaces.
 * Weddings are unique in WedLinker, there would not be any duplicated Weddings.
-* Contacts can be assigned to the Wedding using the [assign-wedding](#assign-contact-to-a-wedding--assign-wedding) command.
+* Contacts can be assigned to the Wedding using the [assign-wedding](#assign-contact-to-a-wedding-assign-wedding) command.
+
+<box type="warning" seamless>
+<details open>
+<summary><strong>Caution:</strong></summary>
+<br>
 
-### Assign contact to a Wedding : `assign-wedding`
+
+- Weddings with single or multiple spaces in their names are considered distinct and treated as separate weddings. For example:
+    - <span class="preserve-whitespace">O'Hare's Wedding vs. O'Hare's  Wedding</span>
+</details>
+</box>
+
+##### Assign contact to a Wedding : `assign-wedding`
 
 Assigns a contact to a `Wedding`.
 
-Format: `assign-wedding INDEX w/WEDDINGNAME`
+Format: `assign-wedding INDEX w/WEDDINGNAME [p1/] [p2/]`
 
 * Assigns a specified contact to the `Wedding` based on their `INDEX`.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​.
-* The `Wedding` must exists in WedLinker before it can be assigned.
+* The `WeddingName` must be that of a `Wedding` that exists in WedLinker before it can be assigned.
 * If the `Wedding` does not exist, you can use `f/` to force the creation and assignment of the `Wedding`.
```

Comment:
Nit but i feel like saying "If the `Wedding` with that `WeddingName` does not exist..." may be clearer? Don't have to take this suggestion if you feel like the current one is okay though